src/components/Tabs.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: (tab: Tab) => void,
+};
+
+export const Tabs: React.FC<Props> = ({ tabs, selectedTabId, onTabSelected }) => (
+  <ul className="nav nav-pills mb-3">
+    {
+      tabs.map(tab => (
+        <li key={tab.id} className="nav-item">
+          {selectedTabId === tab.id
+            ? (
+              <button
+                type="button"
+                onClick={() => onTabSelected(tab)}
+                className="nav-link active"
+              >
+                {tab.title}
+              </button>
+            )
+            : (
+              <button
+                type="button"
+                onClick={() => onTabSelected(tab)}
+                className="nav-link"
+              >
+                {tab.title}
+              </button>
+            )}
```

Comment:
buttons are the same, add class `active` with condition rendering and remove condition rendering for buttons